### ISSUE TITLE
Add Brave browser

### ISSUE THREAD

**Issue by alexmironof** [gopasspw/gopassbridge#109]
I actually managed to run gopassbridge inside Brave with manually copied chromium manifest file :sweat_smile:

**Comment by martinhoefling** [comment]
Great, Brave is pretty exotic. Feel free to add the paths for Brave as PR or link to the corresponding documentation so that we can add it.

**Comment by Pharb** [comment]
From looking at the documentation it seems that you can just install extensions from the Chrome Web Store: https://support.brave.com/hc/en-us/articles/360017909112-How-can-I-add-extensions-to-Brave-

Would be interesting to know, what exactly we would need to change? Shouldn't it just work?

**Comment by alexmironof** [comment]
@Pharb  yes, brave fully supports chrome web store as it is based on latest chromium.
On Ubuntu brave config path is `~/.config/BraveSoftware/Brave-Browser/NativeMessagingHosts`

**Comment by martinhoefling** [comment]
So this we need to implement in  gopass (gopass jsonapi configure). @dominikschulz do you prefer to keep issues like this here or shall we transfer it to gopass?

**Comment by dominikschulz** [comment]
Whatever works best for you.
